doc/reviewing_PR_guide.md

```diff
@@ -0,0 +1,91 @@
+
+Remember, *these are suggestions for how to review a PR*, not the rules for PR inclusion.  A PR can break any of these and it may not be a problem, but in general if a reviewer notices that these aren't met, it's a good sign that a PR might need to be looked at by someone higher in the project.
+
+## 1. Focused on a single area.
+
+PRs must focus on a single area of changes.  While it's tempting to "just fix this other thing while I'm in here", this makes a PR messy to read, and even worse, if you add an unexpected bug, makes that bug much harder to track down.  This is a very important rule, and I can't think of any real excuse for violating it, even though I understand the temptation very well and am guilty of it myself.  Note that this doesn't mean that a PR can't reach into a lot of files, or change a lot of things at once.  Sometimes that is inevitable.  On top of that, we don't want to have a PR add code for features that aren't used, most of the time, so we do recommend a PR try to bring everything to a functional level and implement at least a bit of it all at once.
+
+In general, we would prefer even "overhaul" PRs to break their changes into bite-sized chunks where possible.  A PR rewriting how monster attacks work, with associated JSON changes might, for example: first implement the new attack code but leave the legacy attack code intact, and only change attacks on classic zombies; then, a follow-up PR might change attacks in all other zombies; then, a follow-up PR might change attacks on other monsters; then, a final PR might remove the legacy attack code.  Doing it like this gives us time to track the changes and notice problems in between PRs, and lets us review *both* the code for the new attacks *and* the balance of how monsters' attacks are changing.  If this were all done in a single 3000-line PR, it would be nearly impossible to carefully comb through each changed monster and make sure their new attack stats are appropriate.
+
+As with all the other rules, this does NOT mean you should immediately close your large PR and give up.  If you have a large PR and can't see any reasonable way to break it up, there's a good chance it's still fine; it just needs a higher-up person to review it.
+
+## 2. Length restrictions
+
+The length restrictions are just a suggestion, but an important one.  If a PR goes longer than this, there's a high chance it violates rule 1.  If it does not violate rule 1 and has good reason for being very long (eg. it's a simple automated change to a ton of documents), then this one might be skipped.  If the changes are very tidily broken into organized commits, and those commits are not squashed during merge, then it might also be safe to ignore this one.  Mapgen changes are also pretty much always exempt because maps are huge... but if you have a lot of other changes, it might be a good idea to do the maps in a separate PR first.
```

Comment:
You call those guidelines "rules" again here and in the next line.

Reply:
```suggestion
The length restrictions are just a suggestion, but an important one.  If a PR goes longer than this, there's a high chance it violates point 1.  If it does not violate point 1 and has good reason for being very long (eg. it's a simple automated change to a ton of documents), then this one might be skipped.  If the changes are very tidily broken into organized commits, and those commits are not squashed during merge, then it might also be safe to ignore this one.  Mapgen changes are also pretty much always exempt because maps are huge... but if you have a lot of other changes, it might be a good idea to do the maps in a separate PR first.
```